Assets/PatchKit Patcher/Scripts/UI/Languages/TextTranslator.cs

```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+
+namespace PatchKit.Unity.UI.Languages
+{
+    [RequireComponent (typeof(Text))]
```

Comment:
Unnecessary space before the parenthesis.

Reply:
done